src/SourceLocator/SourceStubber/ReflectionSourceStubber.php

```diff
@@ -76,21 +76,20 @@ public function __construct()
      */
     public function generateClassStub(string $className): ?StubData
     {
-        /** phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName */
-        $isEnum = function_exists('enum_exists') && \enum_exists($className, false);
-        /** phpcs:enable */
-
         if (
             ! (
             class_exists($className, false)
             || interface_exists($className, false)
             || trait_exists($className, false)
-            || $isEnum
             )
         ) {
             return null;
         }
 
+        /** phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName */
```

Comment:
I suppose importing the function won't work due to PHP 8.0 and static analysis?

Reply:
It works but PHPStan does not like `use function` for missing functions so this is hack for PHPStan. Consulted with @ondrejmirtes